README.md

```diff
@@ -35,6 +35,20 @@ Therefore we introduced 2 different mechanisms:
 1. The [YAML metadata describing](Contribute/Adding-Metadata/index.md) `versionFrom` and `versionTo`.
 2. The possibility [to add multiple files about the same topic](Contribute/File-Naming-Conventions/index.md).
 
+# Previewing rendered output locally
+
+There is an experimental project which supports rendering the documentation locally.
+
+You can install the tool from [NuGet](https://www.nuget.org/packages/Umbraco.Docs.Preview.App/) by issuing the following command.
+
+```bash
+$ dotnet tool install --global Umbraco.Docs.Preview.App
+```
+
+Or check out the source at: [https://github.com/umbraco/UmbracoDocs.Preview](https://github.com/umbraco/UmbracoDocs.Preview).
+
+Once installed, navigate to your local clone of the UmbracoDocs repository and run the command `umbracodocs`. This will start a local webserver (the URL will be shown), which you can open in your browser and view the rendered documentation.
```

Comment:
```suggestion
Once installed, navigate to your local clone of the UmbracoDocs repository and run the command `umbracodocs`. This will start a local webserver that's listening on `http://localhost:5000` and `https://localhost:5001` by default (the actual URLs will be shown), which you can open in your browser to view the rendered documentation.
```